modules/samples/oauth2/playground2/src/main/webapp/index.jsp

```diff
@@ -1,11 +1,11 @@
 <%@page import="org.wso2.sample.identity.oauth2.OAuth2Constants"%>
-<%
-    
-    session.removeAttribute(OAuth2Constants.OAUTH2_GRANT_TYPE);
-    session.removeAttribute(OAuth2Constants.ACCESS_TOKEN);
-    session.removeAttribute(OAuth2Constants.CODE);
+<%--<%--%>
```

Comment:
What's the reason behind commenting ? If this code is no longer needed, remove it. If yo think this will have a use in future, add comment with an explanation

Reply:
Fixed in commit: Frontchannel logout functionality added to playground2 app.